components/Searchbar.js

```diff
@@ -0,0 +1,151 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { useRef, useState, useEffect } from 'react';
+
+export default function Searchbar({ books }) {
```

Comment:
Rename file to SearchBar, and rename this function to SearchBar. It is two words.